app/models/legal_aid_application.rb

```diff
@@ -29,6 +29,8 @@ class LegalAidApplication < ApplicationRecord # rubocop:disable Metrics/ClassLen
   has_many :ccms_submissions, class_name: 'CCMS::Submission', dependent: :destroy
   has_one :most_recent_ccms_submission, -> { order(:created_at) }, class_name: 'CCMS::Submission'
   has_one :vehicle, dependent: :destroy
+  has_many :application_scope_limitations, dependent: :destroy
+  has_many :scope_limitations, through: :application_scope_limitations, dependent: :destroy
```

Comment:
I'm a bit worried about the `dependent: :destroy` here.
I wonder if it means that when we delete the `legal_aid_application` it will also delete the `scope_limitations`.
And we don't want that do we?

Reply:
No we don't!